lib/explorer/data_frame.ex

```diff
@@ -3052,14 +3052,34 @@ defmodule Explorer.DataFrame do
 
   ## Options
 
-    * `:nils` - `:first | :last`.
-      Determines if `nil`s get sorted first or last in the result.
-      Default is `:last`.
+    * `:nils` - `:first` or `:last`.
+      By default it is `:last` if direction is `:asc`, and `:first` otherwise.
 
-    * `:stable` - `boolean()`.
+    * `:parallel` - boolean.
+      Whether to parallelize the sorting.
+      By default it is `false`.
+
+      > #### Notice {: .notice}
+      >
+      > The `:parallel` option is applicable under one condition: all lazy series must reduce to
+      > `op: :column`. E.g.
```

Comment:
I had trouble explaining/wording this notice.

Reply:
I like this way with the example!

Reply:
Thanks! I ended up dropping it for simpler wording: [3c77fe5](https://github.com/elixir-explorer/explorer/pull/771/commits/3c77fe564f05a0f12a448e1e83ace922537259a6)

But if enough users say they're confused, we can resurrect that notice.